processor/spanmetricsprocessor/config_test.go

```diff
@@ -22,96 +22,79 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"go.opentelemetry.io/collector/component"
-	"go.opentelemetry.io/collector/exporter/otlpexporter"
-	"go.opentelemetry.io/collector/otelcol/otelcoltest"
+	"go.opentelemetry.io/collector/confmap/confmaptest"
 	"go.opentelemetry.io/collector/pdata/pmetric"
-	"go.opentelemetry.io/collector/processor/batchprocessor"
-	"go.opentelemetry.io/collector/receiver/otlpreceiver"
-
-	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/jaegerexporter"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver"
 )
 
-func TestLoadConfig(t *testing.T) {
+func TestLoadConfig1(t *testing.T) {
+	t.Parallel()
+
 	defaultMethod := "GET"
-	testcases := []struct {
-		configFile                  string
-		wantMetricsExporter         string
-		wantLatencyHistogramBuckets []time.Duration
-		wantDimensions              []Dimension
-		wantDimensionsCacheSize     int
-		wantAggregationTemporality  string
-		wantMetricsFlushInterval    time.Duration
+	tests := []struct {
+		id           component.ID
```

Comment:
What do you think about adding a `name` or `description` attribute to each test case that gives a short description of the test's aims? Alternatively, we could add descriptions above each of the components in `config.yaml` that explains the features of that particular config, much like what was present before in `config-2-pipelines.yaml`, etc.

The reason for this suggestion is because `spanmetricsprocessor` configuration is a little unique (AFAIK) whereby a metrics exporter needs to be configured.